### PR TITLE
Support background.page in extension manifest

### PR DESCRIPTION
**Pull request by jhen0409**
For [chrome-react-perf](https://github.com/crysislinux/chrome-react-perf/blob/master/chrome/manifest.prod.json#L25) support.

### CODE REVIEW

File: lib/browser/chrome-extension.js [hunk 1]
```diff
@@ -309,7 +314,11 @@ app.once('ready', function () {
     const manifest = manifestMap[parsed.hostname]
     if (!manifest) return callback()
 
-    if (parsed.path === '/_generated_background_page.html' &&
+    const backgroundPath = '/' +
+      (manifest.background && manifest.background.page ||
```
Comment (zcbenz) [inline lib/browser/chrome-extension.js]: Can you save the name of background page to `backgroundPages` so we don't have to test `background.page` in all places?

Comment (jhen0409) [inline lib/browser/chrome-extension.js]: @zcbenz updated. 👍

File: lib/browser/chrome-extension.js [hunk 2]
```diff
@@ -72,21 +72,28 @@ const backgroundPages = {}
 const startBackgroundPages = function (manifest) {
   if (backgroundPages[manifest.extensionId] || !manifest.background) return
 
-  const scripts = manifest.background.scripts.map((name) => {
-    return `<script src="${name}"></script>`
-  }).join('')
-  const html = new Buffer(`<html><body>${scripts}</body></html>`)
+  let html
+  let name = '_generated_background_page.html'
```
Comment (kevinsawicki) [inline lib/browser/chrome-extension.js]: Might be cleaner to set this in the `else` block after line `80` and just initialize to undefined like the `html` variable for consistency.

**Comment by kevinsawicki** [comment]
Thanks for fixing this 👍 🚢

**Pull request merged**